### ISSUE TITLE
A11y_AzureDataStudio_ManageAccessForFile_ScreenReader:"OWNERS ", "ALL OTHERS "and " NAMED USERS AND GROUPS" is not marked as table.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7806]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG4.2.3;#WCAG4.1.2;#WCAG4.2.1;#DesktopApp;#Win32;#NVDA;
**Environment Details: **
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit:6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
**Additional Details:**
Tool Name : NVDA
**Steps to Reproduce:**
Install and Launch the application the azure data studio application.
2.Connet to the servers with the required credentials.
3.Navigate to the control panel and click on the “Jupyter books” icon.
4.Navigate till HDFS folder and right click on it and click on manage access.
Start NVDA and observe the announcement.
**Actual:**
Issue 1.NVDA is not announcing the "OWNERS ", "ALL OTHERS "and " NAMED USERS AND GROUPS" as table as there is no table mark up for that.
Issue 2.Table name is not announced by NVDA.
Issue 3.Header reference is  not announced with the header.
Issue 4.  Header reference is not announced along with the row content.
**Expected:**
Issue 1.NVDA is not announcing the "OWNERS ", "ALL OTHERS "and " NAMED USERS AND GROUPS" as table as there should be table mark up.
Issue 2.Table name is should announced by NVDA.
Issue 3.Header reference is  should announced with the header.
Issue 4.  Header reference should announce along with the row content.
**User Impact:**
Screen Reader user will not able to relate with the content.
**Attachment for Reference:**

[7806_A11y_AzureDataStudio_ManageAccessForFile_ScreenReader5.pptx](https://github.com/microsoft/azuredatastudio/files/3750228/7806_A11y_AzureDataStudio_ManageAccessForFile_ScreenReader5.pptx)

Does this issue occur when all extensions are disabled?: Yes
